samples/RhythmGame/src/main/cpp/audio/Player.h

```diff
@@ -27,29 +27,36 @@
 #include <android/asset_manager.h>
 
 #include "RenderableAudio.h"
+#include "DataSource.h"
 
-class SoundRecording : public RenderableAudio{
+class Player : public RenderableAudio{
 
 public:
-    SoundRecording(const int16_t *sourceData, int32_t numFrames)
-            : mData(sourceData)
-            , mTotalFrames(numFrames)
+    /**
+     * Construct a new Player from the given DataSource. Note that the Player does *not* take
+     * ownership of the source so it's up to the caller to delete the source after it's no longer
```

Comment:
Document that Multiple players can share the same DataSource

Reply:
Done